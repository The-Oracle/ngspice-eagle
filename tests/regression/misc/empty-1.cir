check that we can survive emptiness

* (exec-spice "ngspice -b %s")

* Nothingness, No circuit elements at all.
*   We have only the implicit node "0"
* Checks whether we can live with a
*   zero times zero circuit matrix

.control
op
tran 0.1m 1m
ac lin 3 1kHz 2kHz
echo "TEST: done"
quit 0
.endc

.end
