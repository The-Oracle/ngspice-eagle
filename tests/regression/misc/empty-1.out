
Circuit: check that we can survive emptiness

Doing analysis at TEMP = 27.000000 and TNOM = 27.000000



No. of Data Rows : 1
Doing analysis at TEMP = 27.000000 and TNOM = 27.000000


Initial Transient Solution
--------------------------

Node                                   Voltage
----                                   -------



No. of Data Rows : 59
Doing analysis at TEMP = 27.000000 and TNOM = 27.000000



No. of Data Rows : 3
TEST: done
